core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/dto/ScriptDtoService.java

```diff
@@ -53,47 +45,21 @@ public Script convertToEntity(ScriptDto scriptDto) {
                         .collect(Collectors.toList()));
     }
 
-    public ScriptDto convertToDto(Script script) {
-        return new ScriptDto(script.getName(), script.getDescription(),
-                scriptVersionDtoService.convertToDto(script.getVersion()),
-                script.getParameters().stream().map(scriptParameterDtoService::convertToDto).collect(Collectors.toList()),
-                script.getActions().stream().map(scriptActionDtoService::convertToDto).collect(Collectors.toList()),
-                script.getLabels().stream().map(scriptLabelDtoService::convertToDto).collect(Collectors.toList()),
-                null,
-                null);
-    }
-
     @Override
-    public List<ScriptDto> getAll() {
-        return getAll(new ArrayList<>());
+    public Page<ScriptDto> getAll(Pageable pageable, List<String> expansions, boolean isLatestOnly) {
+        if (expansions == null) expansions = new ArrayList<>();
```

Comment:
See comment in repository

Reply:
As I bought back default value, should I delete that line 50 ?

Reply:
yes